tests/unit/helpers/format-date-test.ts

```diff
@@ -26,7 +26,9 @@ module('format-date', function (hooks) {
 
   test('should render empty string for a null value', async function (assert) {
     assert.expect(1);
-    await render(hbs`{{format-date null}}`);
+    await render(hbs`
+      {{~! @glint-expect-error ~}}
+      {{format-date null}}`);
```

Comment:
If there are tests that explicitly verify the behavior of `format-date` with a `null` input, should the types allow for that?

Reply:
The underlying library's (`@formatjs/intl`) types don't allow `null`.